src/lib_ccaligner/params.cpp

```diff
@@ -6,55 +6,30 @@
 
 #include "params.h"
 
-const std::string currentTime()
-{
-    time_t now = time(0);
-    struct tm tstruct = * localtime(&now);
-    char localTime[32]; //being generous
-    strftime(localTime, sizeof(localTime), "%d-%m-%Y-%H-%M-%S", &tstruct);
-    return std::string(localTime);
+#include <ctime>
+
+// Default paths.
+namespace {
+    constexpr auto defaultModelPath = "model/";
+    constexpr auto defaultLmPath = "tempFiles/lm/complete.lm";
+    constexpr auto defaultDictPath = "tempFiles/dict/complete.dict";
+    constexpr auto defaultFsgPath = "tempFiles/fsg/";
+    constexpr auto defaultPhoneticLmPath = "model/en-us-phone.lm.bin";
 }
 
 
-Params::Params()
+Params::Params() noexcept
```

Comment:
Using initialiser's list is a good idea, @alexbrt suggested it to me few days ago as well. Would you like to use it at other relevant places (if required) too?

The only minute problem is that - currently it's not very legible. How about aligning them in each line? I know it's going to take some space, but it's OK.